playbooks/inventory/vagrant.py

```diff
@@ -17,9 +17,18 @@ def parse_args():
 
 
 def list_running_hosts():
+    hosts = {}
+
+    try:
+        status = subprocess.call(["which","vagrant"])
```

Comment:
if you use ```check_call``` then you don't need to check the exit code since it'll raise an exception.